rotkehlchen/tests/api/test_nfts.py

```diff
@@ -948,3 +949,84 @@ def mock_uniswap_v3_balances(*_args, **_kwargs) -> dict[str, Any]:
+                    collection=Collection(
+                        name='ENS: Ethereum Name Service',
+                        banner_image=None,
+                        description='Ethereum Name Service (ENS) domains are secure domain names for the decentralized world. ENS domains provide a way for users to map human readable names to blockchain and non-blockchain resources, like Ethereum addresses, IPFS hashes, or website URLs. ENS domains can be bought and sold on secondary markets.',  # noqa: E501
+                        large_image='https://i.seadn.io/gae/BBj09xD7R4bBtg1lgnAAS9_TfoYXKwMtudlk-0fVljlURaK7BWcARCpkM-1LGNGTAcsGO6V1TgrtmQFvCo8uVYW_QEfASK-9j6Nr?w=500&auto=format',  # noqa: E501
+                        floor_price=FVal(0.00098),
+                    ),
+                ),
+            ],
+            TEST_ACC4: [
+                NFT(
+                    token_identifier='_nft_0x57f1887a8bf19b14fc0df6fd9b2acc9af147ea85_26612040215479394739615825115912800930061094786769410446114278812336794170041',  # noqa: E501
+                    background_color=None,
+                    image_url='https://openseauserdata.com/files/3f7c0c7d1ba51e61fe05ef53875f9f7e.svg',  # noqa: E501
+                    name='yabir.eth',
+                    external_link='https://app.ens.domains/name/yabir.eth',
+                    permalink='https://opensea.io/assets/ethereum/0x57f1887a8bf19b14fc0df6fd9b2acc9af147ea85/26612040215479394739615825115912800930061094786769410446114278812336794170041',  # noqa: E501
+                    price_eth=FVal(0.00098),
+                    price_usd=FVal(1.2379458),
+                    collection=Collection(
+                        name='ENS: Ethereum Name Service',
+                        banner_image=None,
+                        description='Ethereum Name Service (ENS) domains are secure domain names for the decentralized world. ENS domains provide a way for users to map human readable names to blockchain and non-blockchain resources, like Ethereum addresses, IPFS hashes, or website URLs. ENS domains can be bought and sold on secondary markets.',  # noqa: E501
+                        large_image='https://i.seadn.io/gae/BBj09xD7R4bBtg1lgnAAS9_TfoYXKwMtudlk-0fVljlURaK7BWcARCpkM-1LGNGTAcsGO6V1TgrtmQFvCo8uVYW_QEfASK-9j6Nr?w=500&auto=format',  # noqa: E501
+                        floor_price=FVal(0.00098),
+                    ),
+                ),
+            ],
+        }
+        return data, 2
+
+    get_all_nft_data_patch = patch('rotkehlchen.chain.ethereum.modules.nft.nfts.Nfts._get_all_nft_data', side_effect=mock_get_all_nft_data)  # noqa: E501
+
+    with get_all_nft_data_patch:
+        response = requests.get(  # Populate the NFTs
+            api_url_for(
+                rotkehlchen_api_server,
+                'nftsbalanceresource',
+            ),
+            json={'async_query': False, 'ignore_cache': True},
+        )
+        result = assert_proper_response_with_result(response)
```

Comment:
This is also querying `get_v3_balances`. Either mock it to be a no-op or make a cassette. I would prefer to make it a no-op

Reply:
This is not needed because uniswap module is not activated